MultiSensor 7 parameter settings update

Aeotec-ccheng opened the pull request:
Separated MS6 and MS7 configuration SET settings (byte size difference and allowable range settings)

Aeotec-ccheng committed:
MultiSensor 7 parameter settings update

Separated MS6 and MS7 configuration SET settings (byte size difference).

Edited the available settings for 30 - 12 hours rather than 20 seconds to 24 hours.

Aeotec-ccheng commented:
@greens - could you take a look if this is good enough to merge for parameter setting byte size changes? I've tested this one out and it works well with MS6 and MS7 on my side.

greens commented on devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy:
```diff
@@ -74,12 +74,12 @@ metadata {
 
 		input "reportInterval", "enum", title: "Report Interval", description: "How often the device should report in minutes",
-			options: ["8 minutes", "15 minutes", "30 minutes", "1 hour", "6 hours", "12 hours", "18 hours", "24 hours"]
+			options: ["1 minute", "2 minutes", "3 minutes", "4 minutes", "8 minutes", "15 minutes", "30 minutes", "1 hour", "6 hours", "12 hours"]
```
we don't want to allow the interval this low

we find that customers get very concerned when their batteries die quickly

Aeotec-ccheng replied:
Even when set to 1 minute, the lowest setting is 4 minutes on battery power, Wakeup interval generally sets the minimum most setting so it generally won't change unless wakeup interval is set to 240 seconds which is the lowest setting.

If Wakeup Interval = 3600
If Parameter 111 = 60

Sensors will still report every 60 seconds. 

Only on USB power is this setting unlocked to as low as once per a second (or in this case 1 minute) since wakeup does not play a hand in this anylonger as a repeater device.

greens replied:
ok

greens commented on devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy:
```diff
@@ -398,46 +398,96 @@ def ping() {
+            		//when it is waken up.
+            		request << zwave.configurationV1.configurationSet(parameterNumber: 40, size: 1, scaledConfigurationValue: 1)
+			break
+	}
```
a lot of these parameters are the same across models, couldn't you build a `request` list that has all the parameters in common and only separate out those parameters that are different inside the switch statement?

Aeotec-ccheng replied:
Got it, i moved a bunch of the same lines from the switch statement outside if they are the same between MS6 and MS7. I left Parameter 3 since they have different range supports.

Aeotec-ccheng committed:
Updated Configuration

- Added EU Fingerprint for EU MS6
- Reduced coding in configuration and moved out similar settings outside of switch case.

greens commented on devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy:
```diff
@@ -398,46 +399,74 @@ def ping() {
+        request << zwave.configurationV1.configurationSet(parameterNumber: 101, size: 1, scaledConfigurationValue: 240)
+
+        //association group 2
+        request << zwave.configurationV1.configurationSet(parameterNumber: 102, size: 1, scaledConfigurationValue: 1)
```
nit: looks like you've accidentally done spaces instead of tabs here

Aeotec-ccheng replied:
I did a quick change to the tabs, let me know if there is anything else that i need to fix.

Aeotec-ccheng committed:
Updated Configuration space to tabs

Changed spaces to tabs.

Aeotec-ccheng reviewed (commented):
accidentally started this review/comment - just needs to be deleted (sorry)
